### PR TITLE
[On hold] Bumps gRPC

### PR DESCRIPTION
**Pull request by juanpedromoreno**
## What this does?

Bumping up grpc, according to the [grpc-java security guide](https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty) and the code related to [grpc 1.26.0](https://github.com/grpc/grpc-java/blob/v1.26.0/build.gradle#L147).

Closes https://github.com/higherkindness/mu-scala/pull/727
Closes https://github.com/higherkindness/mu-scala/pull/696

## Checklist

- [x] Reviewed the diff to look for typos, println and format errors.
- [ ] Updated the docs accordingly.

### CODE REVIEW

File: project/ProjectPlugin.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ object ProjectPlugin extends AutoPlugin {
       val fs2: String                = "2.2.1"
       val fs2Grpc: String            = "0.6.0"
       val fs2Kafka: String           = "0.20.2"
-      val grpc: String               = "1.24.2"
+      val grpc: String               = "1.26.0"
```
Comment (fedefernandez) [inline project/ProjectPlugin.scala]: I was taking a look to this yesterday, I'm not sure if we should wait until the next `fs2-grpc` release

https://mvnrepository.com/artifact/org.lyranthe.fs2-grpc/java-runtime_2.13/0.6.0

Comment (BenFradet) [reply project/ProjectPlugin.scala]: also I'm not sure about 1.26.0, it's still not mentioned on https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Comment (juanpedromoreno) [reply project/ProjectPlugin.scala]: @fedefernandez yeah, makes sense.

@BenFradet seems like 1.25.x and 1.26.x are using the same netty dependencies: https://github.com/grpc/grpc-java/blob/v1.26.0/build.gradle#L147

**Comment by fedefernandez** [comment]
FYI https://github.com/fiadliel/fs2-grpc/pull/168

**Comment by juanpedromoreno** [comment]
https://github.com/fiadliel/fs2-grpc/releases/tag/v0.7.0 :tada: 

I'll resume this WIP tomorrow

**Pull request closed without merging**